cclib/method/volume.py

```diff
@@ -204,6 +204,8 @@ def scinotation(num):
     return ("%sE%s%s" % (broken[0], sign, broken[1][-2:])).rjust(12)
 
 
+

```

Comment:
Unnecessary extra lines.

Reply:
I've reformatted the file with `black`. Thanks!